.travis.yml

```diff
@@ -19,10 +19,8 @@ matrix:
     - env: PGVERSION=9.6
     - env: PGVERSION=10
     - env: PGVERSION=11
-  allow_failures:
-    - env: PGVERSION=11
 before_install:
-  - git clone -b v0.7.6 --depth 1 https://github.com/citusdata/tools.git
+  - git clone -b pg11-patch --depth 1 https://github.com/citusdata/tools.git
```

Comment:
Did you open a pull request for the tools change? I can merge it (I need to fix the Homebrew error anyhow).